bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/MadokaValue.java

```diff
@@ -25,6 +26,21 @@
 @NonNullByDefault
 public class MadokaValue {
 
+    public enum Endianness {
+        LITTLE_ENDIAN(ByteOrder.LITTLE_ENDIAN),
+        BIG_ENDIAN(ByteOrder.BIG_ENDIAN);
+
+        private ByteOrder order;
+
+        private Endianness(ByteOrder order) {
+            this.order = order;
+        }
+
+        public ByteOrder getOrder() {
+            return this.order;
+        }
+    }
```

Comment:
Why do you need an enum for this when you can just use the `ByteOrder` class directly instead?

Reply:
Yep - fixed.